rosdep/base.yaml

```diff
@@ -2901,6 +2905,7 @@ libvtk-java:
   gentoo: [sci-libs/vtk]
   slackware: [VTK]
   ubuntu: [libvtk-java]
+  opensuse: [vtk-java]
```

Comment:
@dirk-thomas Fixed.
And one more question, do you know is there a way to add key for different openSUSE version?